utils/diff_db_json.py

```diff
@@ -40,7 +40,7 @@ def main():
 
             print("Comparing {}".format(a_json_base))
             subprocess.call(
-                "diff -U 3 {} {}".format(a_json_out.name, b_json_out.name),
+                "diff -U 10 {} {}".format(a_json_out.name, b_json_out.name),
```

Comment:
We should consider if there is a way to use `--show-function-line=RE` argument to extract useful information;
```
       -F, --show-function-line=RE
              show the most recent line matching RE
```

Reply:
Example?

Reply:
BTW Not saying we should block merging this pull request on anything here.